src/components/table/table-column-resize/resizable-table-column.component.ts

```diff
@@ -16,11 +16,24 @@ import { BaseResizableTableService, ResizableTableType } from './resizable-table
 })
 export class ResizableTableColumnComponent implements AfterViewInit, OnDestroy {
 
+    /** Show/Hide column resizable handle */
+    private _handleVisible: boolean = true;
+
     /** Disabled the column resizing */
     @Input()
     @HostBinding('class.ux-resizable-table-column-disabled')
     disabled: boolean = false;
 
+    @Input()
+    @HostBinding('class.ux-resizable-table-hide-handle')
```

Comment:
Just noticed that this class binding does not work as intended:
![image](https://user-images.githubusercontent.com/26114101/95466538-58774580-0974-11eb-835f-34ba53fc5ee1.png)